content/logs/processing/_index.md

```diff
@@ -119,6 +117,11 @@ Using the Datadog Agent or the RFC5424 format automatically sets the host value
 * `hostname`
 * `syslog.hostname`
 
+### *source* attribute
+Using the Datadog Agent or the RFC5424 format automatically sets the source value on your logs. However, if a JSON formatted log file includes the following attribute, Datadog interprets its value as the the log's source:
+
+* `ddsource`
+
```

Comment:
You should add a link to https://docs.datadoghq.com/logs/faq/integration-pipeline-reference/ here

Reply:
This is actually not true. Using the RFC5424 format does not automatically set the source value on logs.
You can use SD params to set it (as we explain in the Rsyslog documentation) but it is not automatic.